scripts/analysis/amwg_table.py

```diff
@@ -204,7 +199,8 @@ def amwg_table(adf):
             if len(ts_files) != 1:
                 errmsg =  "Currently the AMWG table script can only handle one time series file per variable."
                 errmsg += f" Multiple files were found for the variable '{var}'"
-                raise AdfError(errmsg)
+                print(errmsg)
+                return
```

Comment:
Might want to update this `errmsg` here to explicitly state that the tables will be skipped.